.github/workflows/build-pelorus.yml

```diff
@@ -7,156 +7,103 @@ on:
     paths:
       - 'exporters/**'
       - 'charts/pelorus/subcharts/exporters/templates/_buildconfig.yaml'
+      - 'charts/pelorus/subcharts/exporters/templates/_imagestream_from_image.yaml'
```

Comment:
couldn't be `'charts/**'`? what special about these 2 files?

Reply:
I am not sure if we want to re-build images on every charts change. E.g. what if we modify only some text file?

Those two files are entry points to the images, one for image from quay and another to create build from the source.

Even if the _buildconfig.yaml references master I still believe we should recreate images as one may want to create new image with same tag.

Other files are really not relevant for the exporter image build process and we should not create them on every change.